src/native/NativeSessionReplay.ts

```diff
@@ -1,4 +1,4 @@
-import type { NativeModule } from 'react-native';
+import { NativeEventEmitter, type NativeModule } from 'react-native';
```

Comment:
Let's split the regular import and the type-only import to avoid issues with older versions of TypeScript that don't support having both type-only and regular imports in the same import statement.

```suggestion
import { NativeEventEmitter } from 'react-native';
import type { NativeModule } from 'react-native';
```